pom.xml

```diff
@@ -50,7 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <project.systemvm.template.location>https://download.cloudstack.org/systemvm</project.systemvm.template.location>
-        <project.systemvm.template.version>4.20.1.0-SNAPSHOT</project.systemvm.template.version>
+        <project.systemvm.template.version>4.20.0.0</project.systemvm.template.version>
```

Comment:
Could you explain @weizhouapache ? I followed https://cwiki.apache.org/confluence/display/CLOUDSTACK/Release+Procedure when setting this version. When should it change?

Reply:
> Could you explain @weizhouapache ? I followed https://cwiki.apache.org/confluence/display/CLOUDSTACK/Release+Procedure when setting this version. When should it change?

Until now, 4.20.1 and main/4.21 will still use the 4.20.0.0 system template on https://download.cloudstack.org/systemvm/4.20/

We can update the systemvm template version in pom.xml when we build new systemvm template later.

The cwiki page needs to be updated I think.

Reply:
> I see, I'll update it then. No problem

good, thanks !